.jenkins/pytorch/win-test.sh

```diff
@@ -64,6 +64,10 @@ run_tests() {
 }
 
 run_tests && assert_git_not_dirty && echo "TEST PASSED"
+if [[ $? > 0 ]]; then
```

Comment:
The internal CI complains that `>` should only be used for strings and `-gt` should be used here. Could you update that please?